api/tox.ini

```diff
@@ -58,7 +58,9 @@ deps =
 commands =
   coverage xml
   sed -i -e 's|filename="|filename="api/|' coverage.xml
-  codecov --disable pycov --commit={env:CI_HEAD_REV} --branch={env:CI_BRANCH_NAME} --pr={env:CI_PR_NUMBER}
+  curl -Os https://uploader.codecov.io/v0.5.0/linux/codecov
```

Comment:
ick...

Reply:
Unfortunately they deprecated the PyPi package, and offer them as binaries now.
We do something similar in [taskgraph](https://github.com/taskcluster/taskgraph/blob/main/taskcluster/ci/fetch/kind.yml#L17)

Reply:
also, FWIW, it looks like codecov isn't even being used in this repo..?